README.md

```diff
@@ -88,17 +88,50 @@ Requirements:
 
 For enhanced usability, we provide a [GNU make][1] compatible [Makefile](Makefile).
 
+The easiest way to build the project is through [Docker][2].
+
+Build using Docker
+---------------
+### Linux, MacOS AMD64
+```shell
+NGINX_VERSION=1.25.2 make build-musl
+```
+
+### MacOS with Apple Silicon 
+```shell
+NGINX_VERSION=1.25.2 ARCH=aarch64 make build-musl
+```
+The resulting nginx module is `.musl-build\ngx_http_datadog_module.so`
+
+The `build-musl` target builds against musl and libc++ a glibc-compatible
+module. The Dockerfile for the docker image used in the process can be found in
+[build_env/Dockerfile](./build_env/Dockerfile).
+
+Build in Local Environment 
+--------------------------
+### Linux
 ```shell
 NGINX_VERSION=1.25.2 make build
 ```
 
-You can set the environment variable `WAF` to `ON` to build an AppSec-supporting
-module:
+### MacOS
+```shell
+MACOS=true NGINX_VERSION=1.25.2 make build
+```
 
+You will need pcre2 to be able to build the module correctly.
+You can install it with:
 ```shell
-WAF=ON NGINX_VERSION=1.25.2 make build
+brew install pcre2
 ```
```

Comment:
PCRE2 is not a requirement on our module. However, the library is required when NGINX is built with some modules, e.g. the HTTP rewrite module. For example building NGINX with `--without-http_rewrite_module` remove the requirement for PCRE.

Your addition is misleading. In the other hand, I agree the current setup with two different build system is not ideal. I would prefer a troubleshooting section for common errors explaining the status and how to solve the issue.